examples/niswitch_control_relays/README.md

```diff
@@ -26,5 +26,11 @@ This is a MeasurementLink example that controls relays using an NI relay driver
 
 This example requires an NI relay driver (e.g. PXI-2567).
 
-By default, this example uses a simulated instrument. To use a physical instrument, edit
-`_constants.py` to specify `USE_SIMULATION = False`.
\ No newline at end of file
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
+- Create a `.env` file in the measurement service's directory or one of its parent directories (such as the root of your Git repository or `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically registered measurement services).
+- Add the following options to the `.env` file to enable simulation via the driver's option string:
```

Comment:
NI-Switch supports an option string, but not via the `niSwitch_InitializeWithTopology` entry point that we are using. It uses separate parameters.

```suggestion
- Add the following options to the `.env` file to enable simulation via the driver's `simulate` and `topology` parameters:
```